app/models/katello/concerns/redhat_extensions.rb

```diff
@@ -41,7 +41,11 @@ def construct_name(family)
       end
 
       def kickstart_repos(host)
-        distros = distribution_repositories(host).where(distribution_bootable: true)
+        distribution_repos = distribution_repositories(host)
+
+        return [] if distribution_repos.empty?
```

Comment:
This might be unlikely to happen, but if `distribution_repositories` returns an empty array we can't call .where on it